llvm/test/CodeGen/RISCV/calling-conv-ilp32e.ll

```diff
@@ -62,6 +69,8 @@ define i32 @callee_float_in_regs(i32 %a, float %b) {
 ; ILP32E-FPELIM-SAVE-RESTORE-NEXT:    mv a0, a1
 ; ILP32E-FPELIM-SAVE-RESTORE-NEXT:    call __fixsfsi
 ; ILP32E-FPELIM-SAVE-RESTORE-NEXT:    add a0, s0, a0
+; ILP32E-FPELIM-SAVE-RESTORE-NEXT:    .cfi_restore ra
+; ILP32E-FPELIM-SAVE-RESTORE-NEXT:    .cfi_restore s0
 ; ILP32E-FPELIM-SAVE-RESTORE-NEXT:    tail __riscv_restore_1
```

Comment:
The CFI directive should emit after the tail call, otherwise debugger  will confused when stop at `add a0, s0, a0`
```suggestion
; ILP32E-FPELIM-SAVE-RESTORE-NEXT:    tail __riscv_restore_1
; ILP32E-FPELIM-SAVE-RESTORE-NEXT:    .cfi_restore ra
; ILP32E-FPELIM-SAVE-RESTORE-NEXT:    .cfi_restore s0
```